packages/react-vapor/src/components/button/examples/ButtonExamples.tsx

```diff
@@ -1,179 +1,103 @@
-                    <div className="form-control">
-                        <Button
-                            enabled={false}
-                            name="Disabled Button link with onClick"
-                            link="http://perdu.com/"
-                            target="_blank"
-                            onClick={() => alert('Hello')}
-                        />
-                    </div>
-                </div>
-                <div className="form-group">
-                    <label className="form-control-label">Button with children Svg</label>
-                    <div className="form-control">
-                        <Button classes={['p1', 'full-content-y']} enabled>
-                            <Svg svgName={'add'} className="icon mod-2x" />
-                        </Button>
-                    </div>
-                </div>
-                <div className="form-group">
-                    <label className="form-control-label">Button with children name and Svg</label>
-                    <div className="form-control">
-                        <Button classes={['p1', 'full-content-y']} name={'Button'} enabled>
-                            <Svg svgName={'add'} className="ml1 icon mod-2x" />
-                        </Button>
-                    </div>
-                </div>
-
-                <div className="form-group mb5">
-                    <label className="form-control-label">Button with custom style</label>
-                    <div className="form-control">
-                        <Button
-                            enabled
-                            style={{
-                                background: 'linear-gradient(to right, blue, black)',
-                                color: 'white',
-                                border: 'none',
-                                borderRadius: 5,
-                            }}
+            <>
+                <Section title="Button Set">
+                    <Section level={2} title="Usability">
+                        <Button enabled={true} name="Enabled button" />
+                        <Button enabled={false} name="Disabled button" />
+                    </Section>
+                    <Section
+                        level={2}
+                        title="Style modifiers"
+                        description="Add the specific class to change the style of the buttons."
+                    >
+                        <Section
+                            level={3}
+                            title="Color modifiers"
+                            description="Change the color of the buttons (with primary, you can either add 'mod-primary' class or set the 'primary' prop to true)."
                         >
-                            I Rock
-                        </Button>
-                    </div>
-                </div>
-            </div>
+                            <div className="form-group">
```

Comment:
Do you really need `form-group` inside your Section? They should have the right spacing already (in theory) :(

Reply:
I think I forgot to delete it, Let me check i'll poke you back